core/nullDC.cpp

```diff
@@ -299,6 +299,10 @@ void LoadSettings()
 	settings.rend.UseMipmaps		= cfgLoadInt("config", "rend.UseMipmaps", 1);
 	settings.rend.WideScreen		= cfgLoadInt("config", "rend.WideScreen", 0);
 	settings.rend.Clipping			= cfgLoadInt("config", "rend.Clipping", 1);
+	settings.rend.
+	    VerticalResolution			= cfgLoadInt("config", "rend.ResolutionPercentage", 100);
+	settings.rend.
+	    HorizontalResolution		= cfgLoadInt("config", "rend.ResolutionPercentage", 100);
 
```

Comment:
Maybe we should clamp the settings on other platforms (Linux) too:
```suggestion
	if (settings.rend.VerticalResolution <= 0 || settings.rend.VerticalResolution > 100)
	  settings.rend.VerticalResolution = 100;
	if (settings.rend.HorizontalResolution <= 0 || settings.rend.HorizontalResolution > 100)
	  settings.rend.HorizontalResolution = 100;
	  
```

Reply:
Actually, clamping in the current solution will work for every GL/GLES platform, so no changes are needed